flow-bridge/src/main/java/org/reactivestreams/ReactiveFlowBridge.java

```diff
@@ -0,0 +1,357 @@
+/************************************************************************
+ * Licensed under Public Domain (CC0)                                    *
+ *                                                                       *
+ * To the extent possible under law, the person who associated CC0 with  *
+ * this code has waived all copyright and related or neighboring         *
+ * rights to this code.                                                  *
+ *                                                                       *
+ * You should have received a copy of the CC0 legalcode along with this  *
+ * work. If not, see <http://creativecommons.org/publicdomain/zero/1.0/>.*
+ ************************************************************************/
+
+package org.reactivestreams;
+
+import java.util.concurrent.Flow;
+
+/**
+ * Bridge between Reactive-Streams API and the Java 9 Flow API.
+ */
+public final class ReactiveFlowBridge {
+    /** Utility class. */
+    private ReactiveFlowBridge() {
+        throw new IllegalStateException("No instances!");
+    }
+
+    /**
+     * Converts a Flow Publisher into a Reactive Publisher.
```

Comment:
Please "Reactive Streams Publisher/Subscriber" everywhere, let's be specific; the putting "Reactive" around imprecisely leads to people misunderstanding it and saying "oh yeah but the Flow one is not Reactive!" ;)